network-api/networkapi/settings.py

```diff
@@ -521,6 +521,8 @@ class DatabasesDict(TypedDict):
 WAGTAIL_USAGE_COUNT_ENABLED = True
 WAGTAIL_I18N_ENABLED = True
 
+WAGTAILIMAGES_EXTENSIONS = ["avif", "gif", "jpg", "jpeg", "png", "webp", "svg"]
```

Comment:
This makes Wagtail formally accept SVG images as well as the other formats described there (the others are the default, I just added SVG).